.changes/unreleased/tinymce-TINY-11383-2024-10-28.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Changing table row type from a CEF cell would not work as expected.
```

people don't know what CEF is.
```suggestion
body: Changing table row type from a contenteditable=false cell would not work as expected.
```